qutip/solver/floquet_bwcomp.py

```diff
@@ -126,13 +134,9 @@ def floquet_wavefunction(f_modes_t, f_energies, f_coeff, t):
     fbasis = FloquetBasis(H, T, args=args, options=options)
     psi_t = fbasis.from_floquet_basis(f_coeff, t)
     """
-    warnings.warn(FutureWarning("`floquet_wavefunction` is deprecated."))
-    return sum(
-        [
-            f_modes_t[i] * np.exp(-1j * f_energies[i] * t) * f_coeff[i]
-            for i in np.arange(1, len(f_energies))
-        ],
-        start=f_modes_t[0] * np.exp(-1j * f_energies[0] * t) * f_coeff[0],
+    raise NotImplementedError(
+        "`floquet_wavefunction` is deprecated. "
```

Comment:
```suggestion
        "`floquet_wavefunction` is not longer provided. "
```
Deprecated implies it is still usable right now, which it is not.